uiowa/uiowa
# [Registrar] Academic calendar skeleton load

This PR makes a new skeleton load block that can be programmatically loaded to show users and screen readers alike that the page is busy loading things.

# How to test
Check the academic calendar to see if when it loads, there is a skeleton before it populates data that shows the user it is loading. For best results, set the block to grab 10 sessions after, and check the "Show previous events" checkbox so that it takes a while to load.
